src/iio_wrapper.cpp

```diff
@@ -693,3 +735,162 @@ int32_t IIOWrapper::decimation_filter()
 
     return valuel;
 }
+
+// --------------------------------------------------------------------------
+
+int IIOWrapper::set_burst_size_selection(int32_t val)
+{
+    long long valuel = val;
+    int ret = iio_device_debug_attr_write_longlong(m_dev,"burst_size_selection", valuel);
+    return ret;
+}
+
+int IIOWrapper::set_burst_data_selection(int32_t val)
+{
+    long long valuel = val;
+    int ret = iio_device_debug_attr_write_longlong(m_dev,"burst_data_selection", valuel);
+    return ret;
+}
+
+int IIOWrapper::set_linear_acceleration_compensation(int32_t val)
+{
+    long long valuel = val;
+    int ret = iio_device_debug_attr_write_longlong(m_dev,"linear_acceleration_compensation", valuel);
+    return ret;
+}
+
+int IIOWrapper::set_point_of_percussion_alignment(int32_t val)
+{
+    long long valuel = val;
+    int ret = iio_device_debug_attr_write_longlong(m_dev,"point_of_percussion_alignment", valuel);
+    return ret;
+}
+
+int IIOWrapper::set_internal_sensor_bandwidth(int32_t val)
+{
+    long long valuel = val;
+    int ret = iio_device_debug_attr_write_longlong(m_dev,"internal_sensor_bandwidth", valuel);
+    return ret;
+}
+
+int IIOWrapper::set_sync_mode_select(int32_t val)
+{
+    long long valuel = val;
+    int ret = iio_device_debug_attr_write_longlong(m_dev,"sync_mode_select", valuel);
+    return ret;
+}
+
+int IIOWrapper::set_sync_polarity(int32_t val)
+{
+    long long valuel = val;
+    int ret = iio_device_debug_attr_write_longlong(m_dev,"sync_polarity", valuel);
+    return ret;
+}
+
+int IIOWrapper::set_data_ready_polarity(int32_t val)
+{
+    long long valuel = val;
+    int ret = iio_device_debug_attr_write_longlong(m_dev,"data_ready_polarity", valuel);
+    return ret;
+}
+
+int IIOWrapper::set_sync_signal_scale(int32_t val)
+{
+    long long valuel = val;
+    int ret = iio_device_debug_attr_write_longlong(m_dev,"sync_signal_scale", valuel);
+    return ret;
+}
+
+int IIOWrapper::set_decimation_filter(int32_t val)
+{
+    long long valuel = val;
+    int ret = iio_device_debug_attr_write_longlong(m_dev,"decimation_filter", valuel);
+    return ret;
```

Comment:
remove ret, simply return the call of the function... remark applies for all changes